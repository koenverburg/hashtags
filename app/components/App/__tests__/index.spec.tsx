/* tslint:disable no-implicit-dependencies */
import React from 'react'
import { App } from '..'
import { shallow } from 'enzyme'

describe('App component', () => {
  it('render a label', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  })
})
