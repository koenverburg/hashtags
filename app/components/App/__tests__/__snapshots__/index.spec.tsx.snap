// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component render a label 1`] = `
<section>
  <h1>
    Hiii static site gen
  </h1>
</section>
`;
